src/supervisor/message_handlers.py

```diff
@@ -221,9 +222,18 @@ def send_message(self, message_details):
         #: Build message object and send it
         mail = Mail(sender_address, recipient_addresses, subject, content)
         mail.attachment = attachments
-        response = self.sendgrid_client.client.mail.send.post(request_body=mail.get())  # pylint: disable=unused-variable
-
-        #: Maybe test the response via response.status_code?
+        try:
+            self.sendgrid_client.client.mail.send.post(request_body=mail.get())
+        except python_http_client.BadRequestsError as e:  # pylint: disable=invalid-name
```

Comment:
**nitpick**: Why not just use `exception` or `error` rather than `e` and get rid of the pylint disable? Just wondering if I'm missing something.

Reply:
A lot of examples use that format, so I assumed that was a commonly-accepted variable (like i, j, etc). However, on further digging, the official docs always use `err` or `exc`, so I'll go through and update.